lib/al/Library/Audio/System/AudioSystem.h

```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+#include <nn/audio.h>
+#include <nn/os.h>
+
+namespace aal {
+class IAudioFrameProcess;
+}
+
+namespace al {
+class AudioResourcePlayerKeeper;
+struct AudioSoundArchiveInfo;
+class SeDataBase;
+class BgmDataBase;
+class AudioInfoListWithParts;
+struct AudioSystemInfo;
+class AudioEffectDataBase;
+class CollisionCodeList;
+class AudioSystemPauseController;
+class AudioSystemNin;
+class AudioBusSendController;
+class AudioFrameProcessMgr;
+class AudioMaximizer;
+struct AudioSystemInitInfo;
+
+class AudioSystem {
+private:
+    al::AudioResourcePlayerKeeper* mAudioResourcePlayerKeeper;
+    al::AudioSoundArchiveInfo** audioResourceLoadingInfo;
+    al::SeDataBase* mSeDataBase;
+    al::BgmDataBase* mBgmDataBase;
+    al::AudioInfoListWithParts* mBgmMusicalInfoList;
+    al::AudioSystemInfo* mAudioSystemInfo;
+    float mMasterVolume;
+    float mDockedVolume;
+    float mUndockedVolume;
+    al::AudioEffectDataBase* mAudioEffectDataBase;
+    al::CollisionCodeList* mMaterialCodeList;
+    al::CollisionCodeList* mMaterialCodePrefixList;
+    bool mIsChangeStereo;
+    bool mIsHeadphones;
+    al::AudioSystemPauseController* mAudioSystemPauseController;
+    al::AudioSystemNin* mAudioSystemNin;
+    al::AudioBusSendController* mAudioBusSendController;
+    al::AudioFrameProcessMgr* mAudioFrameProcessMgr;
+    al::AudioMaximizer* mAudioMaximizer;
+    nn::audio::MemoryPoolType* mMemoryPool;
+    nn::os::SystemEvent mAudioDeviceSwitchNotification;
```

Comment:
Is this correct to be a non-pointer type?

Reply:
Yep, that's why i update NintendoSdk here

Reply:
Alright, ignore this then.